protocol/bin/flow.js

```diff
@@ -472,9 +472,9 @@ function write(typeDefs, project) {
 
 // This file is auto-generated by client/protocol/Makefile.
 // Not enabled: calls need to be turned on in enabled-calls.json
-import engine from '../../engine'
```

Comment:
avoid pulling in engine and engine saga, which pull in some rpc types themselves